publication-doi-commons/src/main/java/no/unit/nva/publication/doi/dto/PublicationStatus.java

```diff
@@ -0,0 +1,22 @@
+package no.unit.nva.publication.doi.dto;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+
+public enum PublicationStatus {
```

Comment:
Create a parsing method that is ignoring case?

Reply:
updated